{"name": "test_invalid_username", "status": "passed", "attachments": [{"name": "Screenshot", "source": "86ecd02b-e5ea-46eb-a61e-546547628609-attachment.png", "type": "image/png"}, {"name": "log", "source": "a8064c98-899f-4ac3-973c-05bf8668d38e-attachment.txt", "type": "text/plain"}], "start": 1687762449921, "stop": 1687762461565, "uuid": "3fef18ce-3c30-4f92-accd-a3729a011c19", "historyId": "aedda9e906929d3f63bbe105eb0fa5d9", "testCaseId": "aedda9e906929d3f63bbe105eb0fa5d9", "fullName": "testCases.test_Login.LoginTest#test_invalid_username", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "@pytest.mark.usefixtures"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "DESKTOP-9PV1AM7"}, {"name": "thread", "value": "11320-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_Login"}]}